Makefile

```
@@ -3,7 +3,7 @@
 # Detect docker compose command (prefer newer 'docker compose' over 'docker-compose')
 DOCKER_COMPOSE := $(shell docker compose version >/dev/null 2>&1 && echo "docker compose" || echo "docker-compose")
 
-.PHONY: help build build-nocache rebuild run stop logs shell clean config edit setup setup-prebuilt update-compose
+.PHONY: help build build-nocache rebuild run stop logs shell clean config edit setup setup-prebuilt update-compose doctor paths
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)